src/components/NftRewards/RewardTier.tsx

```diff
@@ -131,6 +131,7 @@ export function RewardTier({
               : 'bg-smoke-100 dark:bg-slate-600',
             !loading ? 'pt-2' : 'pt-1',
           )}
+          onClick={isSelected ? onRemoveTier : () => null}
```

Comment:
Just wrap in my `stopPropagation` fn (`import { stopPropagation } from 'react-stop-propagation'`)

and use like:
```suggestion
          onClick={isSelected ? stopPropagation(onRemove): undefined}
```

Reply:
Done ser, thanks for the tip. Also applied it to the other `onRemoveTier` call and removed this function.